ecommerce-ui/src/BookingCart/BookingCart.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import RentalProperty from './../RentalProperty/RentalProperty';
+import './BookingCart.css';
+
+
+const propTypes = {
+  rentalProperties: PropTypes.arrayOf(Object)
+}
+
+class BookingCart extends Component {
+
+  constructor(props){
+    super(props);
+    this.state = {
+      total:0
+    }
+    this.calculateTotal = this.calculateTotal.bind(this);
+  }
+
+  calculateTotal(price) {
+    this.setState({
+      total: this.state.total + price
+    });
+  }
+
+  render() {
+    return (
+      <div>
+        <div className="Booking-Cart-container">
```

Comment:
Lines 30-34 should be moved to App.js
BookingCart should be a separate list of booked properties, not all properties listed on the site
There should be a Remove button next to each line in that list.
Imagine a site with thousands of available inventory.   You really can't see what you added to the cart.